frontend/src/Movie/Delete/DeleteMovieModalContentConnector.js

```diff
@@ -16,22 +17,26 @@ function createMapStateToProps() {
 }
 
 const mapDispatchToProps = {
-  deleteMovie
+  deleteMovie,
+  push
 };
 
 class DeleteMovieModalContentConnector extends Component {
 
   //
   // Listeners
-
-  onDeletePress = (deleteFiles, addImportExclusion) => {
+  onDeletePress = (deleteFiles, addImportExclusion, previousMovie) => {
```

Comment:
You are essentially passing `previousMovie` from DeleteMovieModal, to this component then to `DeleteMovieModalContent` and back to here via this parameter... I think you can cut out ever handling it in `DeleteMovieModalContent` and just use the prop that is passed in from `DeleteMovieModal` here.... this avoids any change to `DeleteMovieModalConent` and avoids adding a parameter here.